shared-ui/components/meet-the-team/meet-all-core/MeetAllCore.styles.ts

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+import { P, H3 } from '../../../style/typography';
+import { colors } from '../../../style/colors';
+import { max } from '../../../lib/responsive';
+
+const StyledMeetCoreContainer = styled.div`
+  display: flex;
+  margin: auto;
+  padding: 8em;
+  @media ${max.tabletLg} {
```

Comment:
can we change this to be max.desktop? 8em squishes the contents a bit too much at the screensizes between tabletLg and desktop
<img width="1296" alt="Screenshot 2022-11-19 at 4 17 17 PM" src="https://user-images.githubusercontent.com/59738880/202871916-a950ea2f-a35e-4702-8566-8551a3a9372e.png">

Reply:
vs. 4em
<img width="1235" alt="Screenshot 2022-11-19 at 4 18 12 PM" src="https://user-images.githubusercontent.com/59738880/202871953-2ef64d8f-d104-443f-bb84-203c84ff607c.png">

Reply:
I think you could even go down to 3em for max.desktop!

Actually yeah I think I like 3em more since we have a lot of content to fit at the smaller screensizes

can we change to 
```

 @media ${max.desktop} {
    padding: 3em;
  }
```